jena-fuseki2/examples/config-prefixes.ttl

```diff
@@ -0,0 +1,24 @@
+## Licensed under the terms of http://www.apache.org/licenses/LICENSE-2.0
+
+PREFIX example:  <http://example.org/fuseki/>
+PREFIX fuseki:  <http://jena.apache.org/fuseki#>
+PREFIX rdf:     <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX :        <http://telicent.io/config#>
+PREFIX ja:      <http://jena.hpl.hp.com/2005/11/Assembler#>
+PREFIX tdb2:    <http://jena.apache.org/2016/tdb#>
+
+PREFIX ex: <http://example.org/fuseki/operation/>
```

Comment:
This namespace does not appear to match that actually used later in the code so unclear this example is actually usable?